feature-libs/customer-ticketing/core/facade/customer-ticketing.service.ts

```diff
@@ -19,6 +25,18 @@ import { CustomerTicketingConnector } from '../connectors';
 
 @Injectable()
 export class CustomerTicketingService implements CustomerTicketingFacade {
+  getTicketCategoriesQueryReloadEvents(): QueryNotifier[] | undefined {
+    return [GetTicketCatQueryReloadEvent];
+  }
+  getTicketCategoriesQueryResetEvents(): QueryNotifier[] | undefined {
+    return [GetTicketCatQueryResetEvent];
+  }
+  getTicketAssociatedObjectsQueryReloadEvents(): QueryNotifier[] | undefined {
+    return [GetTicketAssociatedObjectsQueryReloadEvent];
+  }
+  getTicketAssociatedObjectsQueryResetEvents(): QueryNotifier[] | undefined {
+    return [GetTicketAssociatedObjectsQueryResetEvent];
+  }
```

Comment:
Why do these methods return. QueryNotifier[] or undefined?
Shouldn't be enough to return QueryNotifier[], since their implementations are hardcoded to return an array of one element?